src/commonMain/kotlin/exchange.dydx.abacus/processor/router/skip/SkipStatusProcessor.kt

```diff
@@ -0,0 +1,179 @@
+package exchange.dydx.abacus.processor.router.skip
+
+import exchange.dydx.abacus.processor.base.BaseProcessor
+import exchange.dydx.abacus.protocols.ParserProtocol
+import exchange.dydx.abacus.utils.mutable
+
+internal class SkipStatusProcessor(
```

Comment:
none of the methods in this class rely on state in the class, could we delegate to static helper methods and test those?